src/main/java/com/phonebookservice/util/Util.java

```diff
@@ -0,0 +1,30 @@
+package main.java.com.phonebookservice.util;
+
+import java.util.Collection;
+
+public final class Util {
```

Comment:
Anyone will inherit from this class in the future ? Answer is No, so you did it final. 
Any one will create an insance of this object ? Answer is No, why you didn't make it abstract. ?

Reply:
The class Util can be either abstract or final, not both